documents/state_machine.py

```diff
@@ -0,0 +1,62 @@
+from config import config
+from models.db import db
+from models.User import User
+import jwt
+from models.Document import Document
+
+def get_user_by_user_id(user_id):
```

Comment:
why is this try catch still required? 
And if it is, why aren't you using a decorator here?